atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleTableNameMapper.java

```diff
@@ -63,19 +65,19 @@ private String getTableNumberSuffix(
             String fullTableName,
             String truncatedTableName) {
         int tableSuffixNumber = getNextTableNumber(connectionSupplier, truncatedTableName);
-        if (tableSuffixNumber >= 100_000) {
+        if (tableSuffixNumber >= 10_000) {
```

Comment:
this only actually allows 4 digits of tables (_0000 to _9999). This seems like too few, and contradicts the above `SUFFIX_NUMBER_LENGTH`.